### ISSUE TITLE
[Bug]: Turnsile needs a fix and getPageData

### ISSUE THREAD

**Issue by tsundDEV** [waldo-vision/waldo#87]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### What happened?

1. A bug with turnsile where the widget doesn't show up

Console output:
```
sitekey was not provided
```

2. A bug with getPageData where returns code 500

Reference: 
```json
[{"error":{"json":{"message":"Unexpected end of JSON input","code":-32603,"data":{"code":"INTERNAL_SERVER_ERROR","httpStatus":500,"path":"site.getPageData"}}}},{"error":{"json":{"message":"Unexpected end of JSON input","code":-32603,"data":{"code":"INTERNAL_SERVER_ERROR","httpStatus":500,"path":"site.getPageData"}}}},{"error":{"json":{"message":"Unexpected end of JSON input","code":-32603,"data":{"code":"INTERNAL_SERVER_ERROR","httpStatus":500,"path":"site.getPageData"}}}}]
```

### Steps To Reproduce

1. Go to website

### Project

Website

### What browsers are you seeing the problem on?

Firefox, Chrome, Safari, Microsoft Edge

### Relevant log output

```shell
Check references
```

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by ceriddenn** [comment]
This has to do with some kind of issue on prod. Works fine in dev env

**Comment by tsundDEV** [comment]
Fixed in #98 #96 in PR #127
